Lib/venv/__init__.py

```diff
@@ -306,7 +306,7 @@ def setup_python(self, context):
 
             suffixes = ['python', 'python3', f'python3.{sys.version_info[1]}']
             if sys.version_info[:2] == (3, 14):
-                suffixes.append('python𝜋')
+                suffixes.append('𝜋thon')
```

Comment:
Why can't we have both of them? :smile:

Reply:
I think we should decide on one option. An easter egg should be unique, otherwise it'll loose its punch.

Reply:
(I suggest creating the concept of an easter omlet)